ContactDetailsApi/V2/Boundary/Request/ServicesoftFetchContactDetailsRequest.cs

```diff
@@ -10,7 +10,7 @@ public class ServicesoftFetchContactDetailsRequest
         public string PaginationToken { get; set; }
 
         [FromQuery]
-        [Range(typeof(int), "1", "500")]
-        public int PageSize { get; set; } = 500;
+        [Range(1, int.MaxValue)]
+        public int PageSize { get; set; } = int.MaxValue;
```

Comment:
Maybe the XML doc above the endpoint method should be amended to mention that the data pull may timeout with too many values per page requested.

If that's not mentioned, and the max item pull is requested from the start, it may look like our API isn't working. Or is the timeout not an issue anymore?

Reply:
We don't expect it to be an issue even with unlimited page size, but will test on dev to be sure